statistics/interact_with_storage.go

```diff
@@ -236,6 +236,24 @@ func indexStatsFromStorage(reader *StatsReader, row chunk.Row, table *Table, tab
 		if histID != idxInfo.ID {
 			continue
 		}
+		notNeedLoad := lease > 0 &&
```

Comment:
Why add `lease > 0` here? Better to add some comments for this check.

Reply:
Added.

Reply:
`lease > 0` is aligned to `notNeedLoad` conditions of `columnStatsFromStorage`. In UT, `lease` is 0 and all column/index stats are fully loaded.